main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -487,6 +487,7 @@ object JvmOps {
     */
   // TODO: Magnus: Use this in appropriate places.
   def mangle(s: String): String = s.
```

Comment:
Disclaimer: I know nothing about this part of the compiler, so I don't know how important this is.

It looks like some of these replacements will never happen, due to a bad priority order. For example the `$lt` replacement will get rid of al the `<`s in the string, so the `$le` and `$lshift` replacements will never happen.

Is there a reason to have these multi-char manglings rather than single-character ones? From what I can tell, [scala uses single-character matches](https://github.com/scala/scala/blob/2.12.x/src/library/scala/reflect/NameTransformer.scala).

Reply:
I don't know much about it either. But if Scala does something different, I think we can adopt that. They probably worked it out more carefully than we did.

Reply:
> Is there a reason to have these multi-char manglings ...
I think it is to translate the meaning of the operator while Scala is just translating the symbols directly (`&` becoming `$band` vs. `$amp`. Should I change it in the style of Scala?

Reply:
Yes, I think we should go with the Scala solution.